hack/lib/util.sh

```diff
@@ -208,7 +208,12 @@ kube::util::find-binary-for-platform() {
     locations+=("$location");
   done < <(find "${KUBE_ROOT}/bazel-bin/" -type f -executable \
     \( -path "*/${platform/\//_}*/${lookfor}" -o -path "*/${lookfor}" \) 2>/dev/null || true)
-
+  # search for executables for non-GNU versions of find (eg. BSD)
+  while IFS=$'\n' read -r location; do
+    locations+=("$location");
+  done < <(find "${KUBE_ROOT}/bazel-bin/" -type f -perm +111 \
```

Comment:
Does `-perm +111` work for both? If so, should we just use that?

Reply:
It does. I opted to make the change additive to minimize any potential disruption since I imagine this script is used in a wild number of places that I don't know about but I'm happy to change the existing line if you think it's a safe change.

Reply:
GNU no longer supports this syntax https://lists.gnu.org/archive/html/bug-findutils/2013-09/msg00024.html